src/routes/github/configuration/github-configuration-get.ts

```diff
@@ -56,8 +56,11 @@ const installationConnectedStatus = async (
 	reqLog: Logger,
 	gitHubAppId?: number
 ): Promise<MergedInstallation[]> => {
-	const subscriptions = await Subscription.getAllForHost(jiraHost);
+	const subscriptions = await Subscription.getAllForHost(jiraHost, gitHubAppId);
 	const installationsWithSubscriptions = await getInstallations(subscriptions, reqLog, gitHubAppId);
+	await removeFailedConnectionsFromDb(reqLog, installationsWithSubscriptions, jiraHost);
+	reqLog.debug(`Removed failed installations`);
```

Comment:
Should probably just call this logger as this is how it's typically approached in the code when passing req.log

Reply:
Nit: don't need backticks here as we're not passing any values in.